atd-vzv/src/views/nav/SideDrawerContent.js

```diff
@@ -21,9 +21,12 @@ const SideDrawerContent = ({ type }) => {
     @media only screen and (max-width: ${responsive.bootstrapMedium}px) {
       display: none;
     }
-
     .vz-logo {
-      height: ${responsive.headerButtonHeight + responsive.headerLogoOffset}px;
```

Comment:
because of the diff logo dimensions these parameters dont work anymore as a good measure for height

Reply:
cool - those old values look specific to the dimensions of the old logo so I think adding some that are specific to the new one makes sense!